### ISSUE TITLE
rewrite - the required format is <repo name>@<manifest lists digest>

### ISSUE THREAD

**Issue by lioramilbaum** [michaelperel/docker-lock#12]
The current format is not supported in our container images build system.

**Comment by michaelperel** [comment]
Thanks for opening an issue :)

Just to make sure I understand this, given a Dockerfile that looks like:

```
FROM python:3.6
```

Currently, rewrite would make the Dockerfile

```
FROM python:3.6@sha256:767fe8071c0127414efdc30450fd8b847488aeeebae9d987c7611d04bdd9bbf9
```

and you would like the Dockerfile to be

```
FROM python@sha256:767fe8071c0127414efdc30450fd8b847488aeeebae9d987c7611d04bdd9bbf9
```

Is that correct, the tag throws off the build system?

**Comment by lioramilbaum** [comment]
Yes. This is correct. You might want to use a variable to specify the parent image reference format. It enables flexibility. WDYT?

**Comment by michaelperel** [comment]
That sounds great -- I will add a flag to rewrite for flexibility

**Comment by michaelperel** [comment]
@lioramilbaum This feature has been implemented in v0.5.0.

You can now say:
```
docker lock rewrite --exclude-tags
```
to rewrite without tags.

**Comment by lioramilbaum** [comment]
Verified. Thanks.
